beanstalk_worker/worker_dockerfile_lint.py

```diff
@@ -58,8 +60,12 @@ def lint_job_data(job_data):
 
```

Comment:
Check for `err` here before logging the message "Dockerfile Lint check done" (a line above).